lib/install.js

```diff
@@ -242,8 +242,15 @@ Installer.prototype.run = function (_cb) {
     }
   }
 
+  var preinstallSteps = []
   var installSteps = []
   var postInstallSteps = []
+
+  preinstallSteps.push(
```

Comment:
Why introduce a new phase here? Why not just make this the start of the install steps? Either way, this needs to not run of `this.dryrun` is true.